src/commands/resumes/getResume.ts

```diff
@@ -0,0 +1,59 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+import { RESUME_EMBED_COLOUR } from '../../utils/embeds';
+
+class GetResumeCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'get-resume-command',
+      aliases: ['get-resume', 'resume'],
+      group: 'resumes',
+      memberName: 'get-resume',
+      description: 'Get an example resume!',
+      examples: [`${client.commandPrefix}resume web-dev 2`],
+      args: [
+        {
+          key: 'field',
+          prompt: 'The field of the resume (eg web-dev)',
+          type: 'string'
+        },
+        {
+          key: 'term',
+          prompt: 'The work-term the person with the resume will be in during their co-op (eg 3)',
+          type: 'integer'
+        }
+      ]
+    });
+  }
+
+  getResumeEmbed = (field: string, term: string): MessageEmbed => {
+    try {
+      // get link to resume (just first one in the "list" for now)
+      const resumeLink = require('./resumes.json')[field][term][0].url;
+      return new MessageEmbed()
+        .setColor(RESUME_EMBED_COLOUR)
+        .setTitle('Example Resume')
+        .setDescription(`Example resume for ${field} for work term ${term}`)
+        .addFields({
+          name: 'Resume Link',
+          value: resumeLink
+        });
+    } catch (e) {
+      return new MessageEmbed()
+        .setColor(RESUME_EMBED_COLOUR)
+        .setTitle('Example Resume')
```

Comment:
maybe title and description can be part of json, but I dunno what to have em as

Reply:
We could have the embed color be dependent on the field (eg front-end is green, back-end is blue, etc). What do you think

Reply:
totally up to u, idrc abt colour
I'd prefer finishing resume epic before focusing on colour / font size / etc